content/authentication/connecting-to-github-with-ssh/working-with-ssh-key-passphrases.md

```diff
@@ -40,7 +40,7 @@ If your key already has a passphrase, you will be prompted to enter it before yo
 
 ## Auto-launching `ssh-agent` on Git for Windows
 
-You can run `ssh-agent` automatically when you open bash or Git shell. Copy the following lines and paste them into your `~/.profile` or `~/.bashrc` file in Git shell:
+You can run `ssh-agent` automatically when you open bash or Git shell. Copy the following lines and paste them into the `~/.profile` or `~/.bashrc` file found in your `~/git/etc` folder:
```

Comment:
Dropping "Git shell" from this text only makes sense if you forget about all Windows-based users: Outside of the Git Bash (i.e. in a CMD or a PowerShell), `~/.profile` does not make any sense on Windows.

So I would strongly advise against accepting this change as-is.

Reply:
i love you babe like my mom